src/test/resources/application.properties

```diff
@@ -11,4 +11,15 @@ management.endpoint.health.show-details=always
 endpoints.health.sensitive=false
 registry.protocols=http
 
-spring.jpa.hibernate.ddl-auto=none
\ No newline at end of file
+spring.jpa.hibernate.ddl-auto=create
+
+spring.mail.port=587
+spring.mail.test-connection=false
+spring.mail.properties.mail.smtp.auth=false
+spring.mail.properties.mail.smtp.ssl.enable=false
+
+mail.from=amp-dev@ebi.ac.uk
+mail.to=selva@ebi.ac.uk
```

Comment:
Do we really need to include these? Can't we make them Travis environment variables, which Spring can also read? Otherwise we may start receiving lots of spam...